### ISSUE TITLE
new release request

### ISSUE THREAD

**Issue by chenrui333** [tadfisher/pass-otp#134]
👋  it looks like there is no release in the past two years, I wonder if there would be any new release plan in the near future. Thanks!

**Comment by jemadux** [comment]
>  it looks like there is no release in the past two years, I wonder if there would be any new release plan in the near future. Thanks!

if aint broke , dont fix it

**Comment by zeorin** [comment]
Raising this again… some distro packaging happens in response to a new release. There has been new functionality (e.g. https://github.com/tadfisher/pass-otp/pull/117) added to `pass-otp` since the last release that isn't available to most users:

[![Packaging status](https://repology.org/badge/vertical-allrepos/pass-otp.svg)](https://repology.org/project/pass-otp/versions)

**Comment by igxlin** [comment]
Vote it. The steam otp cannot be used in the release version.

**Comment by zeorin** [comment]
@tadfisher No release since almost 4 years :cry: Please consider cutting a new release.

**Comment by zeorin** [comment]
For anyone using Nix, I have overridden the package like so:
```nix
pkgs.pass.withExtensions (exts:
        with exts; [
          (pass-otp.overrideAttrs (oldAttrs:
            let
              perl-pass-otp = with pkgs.perlPackages;
                buildPerlPackage {
                  pname = "Pass-OTP";
                  version = "1.5";
                  src = pkgs.fetchurl {
                    url =
                      "mirror://cpan/authors/id/J/JB/JBAIER/Pass-OTP-1.5.tar.gz";
                    sha256 = "GujxwmvfSXMAsX7LRiI7Q9YgsolIToeFRYEVAYFJeaM=";
                  };
                  buildInputs =
                    [ ConvertBase32 DigestHMAC DigestSHA3 MathBigInt ];
                  doCheck = false;
                };
            in {
              version = "1.2.0.r29.a364d2a";
              src = pkgs.fetchFromGitHub {
                owner = "tadfisher";
                repo = "pass-otp";
                rev = "a364d2a71ad24158a009c266102ce0d91149de67";
                sha256 = "q9m6vkn+IQyR/ZhtzvZii4uMZm1XVeBjJqlScaPsL34=";
              };
              buildInputs = [ perl-pass-otp ];
              patchPhase = ''
                sed -i -e 's|OATH=\$(which oathtool)|OATH=${perl-pass-otp}/bin/oathtool|' otp.bash
                sed -i -e 's|OTPTOOL=\$(which otptool)|OTPTOOL=${perl-pass-otp}/bin/otptool|' otp.bash
              '';
            }))
        ])
```

**Comment by innir** [comment]
@tadfisher Debian freeze is approaching, would be nice to have the new feature in the next stable version...

**Comment by zeorin** [comment]
@tadfisher it's been nearly 5 years since the last release. Please consider cutting a new release. 🙏

**Comment by benaryorg** [comment]
Throwing out a new release either before the current latest commit or after #219 would allow downstream distros to easily package the new version, which has the *otptool* support.
As a Nix user for instance I have to [go out of my way to generate Steam OTP codes](https://git.shell.bsocat.net/infra/commit/?h=unstable&id=b01292c3919d667807308fd93429d68a90c36804) and it would be nice if we could just have that by default in a lot more distros without having to drop stable releases as a concept.
